tools/SetupFlow/DevHome.SetupFlow/Views/RepoConfigReviewView.xaml

```diff
@@ -24,10 +36,67 @@
+            SelectionMode="None"
+            ItemContainerStyle="{ThemeResource ListViewItemStretchStyle}">
+            <ListView.Header>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="40"/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="550" />
+                    </Grid.ColumnDefinitions>
+                    <Border Grid.Column="0" Style="{StaticResource BorderStyle}" BorderThickness="0, 0, 1, 0">
+                        <TextBlock x:Uid="ms-resource:///DevHome.SetupFlow/Resources/RepositoryNameHeader" VerticalAlignment="Center" HorizontalTextAlignment="Left" Padding="15, 0, 0, 0"/>
+                    </Border>
+                    <TextBlock Grid.Column="1" x:Uid="ms-resource:///DevHome.SetupFlow/Resources/RepositoryClonePathHeader" VerticalAlignment="Center" HorizontalTextAlignment="Left" Padding="15, 0, 0, 0"/>
+                </Grid>
+            </ListView.Header>
+            <ListView.ItemTemplate>
+                <DataTemplate x:DataType="models:CloneRepoTask">
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="40"/>
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="550" />
+                        </Grid.ColumnDefinitions>
+                        <Border Grid.Column="0" Style="{StaticResource BorderStyle}" BorderThickness="0, 1, 1, 0">
+                            <Grid ColumnSpacing="5">
+                                <ToolTipService.ToolTip>
+                                    <ToolTip IsEnabled="{x:Bind IsRepoNameTrimmed, Mode=OneWay}" Content="{x:Bind RepositoryOwnerAndName}"/>
+                                </ToolTipService.ToolTip>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="0.1*"/>
+                                    <ColumnDefinition/>
+                                </Grid.ColumnDefinitions>
+                                <FontIcon Grid.Column="0" Width="20" Height="20" FontFamily="{ThemeResource DevHomeFluentIcons}" Glyph="{x:Bind RepositoryToClone.IsPrivate, Mode=OneWay, Converter={StaticResource BoolToGlyphConverter}}"/>
+                                <TextBlock Grid.Column="1" Text="{x:Bind RepositoryOwnerAndName}" VerticalAlignment="Center" HorizontalAlignment="Left" TextTrimming="CharacterEllipsis">
+                                     <i:Interaction.Behaviors>
+                                        <ic:EventTriggerBehavior EventName="IsTextTrimmedChanged">
+                                            <ic:InvokeCommandAction Command="{x:Bind RepoNameTrimmedCommand}"/>
```

Comment:
Since this is a UI logic (not a business logic): "If text is trimmed then show the tooltip", we can implement this behavior in code behind instead of the model/view-model. This will also eliminate the need for the new observable members and relay command.